lib/hyacinth/adapters/digital_object_search_adapter/solr.rb

```diff
@@ -102,6 +78,42 @@ def identifier_to_uids(identifier, opts = {})
         def clear_index
           solr.clear
         end
+
```

Comment:
I would prefer to handle this via YARD documentation than actual access modifier - I don't think there's a concern about unintentional API use, and it makes testing annoying.

Reply:
Maybe even move the method to Solr::Utils?

Reply:
I think of Solr::Utils as having more generic solr functions -- ones that could be used in DigitalObjectSearchAdapter::Solr or TermSearchAdapter::Solr -- and these methods feel like they're tied more to DigitalObjectSearchAdapter, but I'm glad to remove the private designation and use YARD documentation instead.  I'll push an update for that right now.  Good suggestion.  👍